usr/share/rear/conf/GNU/Linux.conf

```diff
@@ -231,3 +231,10 @@ KERNEL_CMDLINE+=" selinux=0"
 CLONE_USERS+=( daemon rpc usbmuxd usbmux vcsa nobody dbus )
 CLONE_GROUPS+=( tty usbmuxd usbmux fuse kvm oinstall dbus )
 
+# Add firmware file for AMD graphics hardware (if present) on TCG Opal pre-boot authentication (PBA) images
+# Cf. https://github.com/rear/rear/issues/2474
+# lspci output is expected to look similar to this:
+#   00:01.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Device 98e4 (rev 81)
+if type -p lspci >/dev/null && lspci | grep --quiet ' VGA .*AMD'; then
+    OPAL_PBA_FIRMWARE_FILES+=( amdgpu )
```

Comment:
`OPAL_PBA_FIRMWARE_FILES+=( amdgpu )` will lead to
`FIRMWARE_FILES=( amdgpu )` but that will not work because
`find /lib*/firmware -ipath 'amdgpu'` will find nothing
cf. the code in build/GNU/Linux/420_copy_firmware_files.sh
and see the FIRMWARE_FILES description in default.conf:
```
# The FIRMWARE_FILES array contains filename globbing patterns
# that are used as '-ipath' arguments for 'find' calls to find
# firmware files in the /lib*/firmware/ directories that will
# get included in the rescue/recovery system for example like
# FIRMWARE_FILES=( '*rtl*' '*nvidia*' '*radeon*' )
```
so what should work is
`OPAL_PBA_FIRMWARE_FILES+=( '*amdgpu*' )`
because with that
`find /lib*/firmware -ipath '*amdgpu*'` finds firmware files
(at least on my openSUSE Leap 15.1 system).

Furthermore if I understand https://github.com/rear/rear/issues/2474 correctly
the `/lib/firmware/amdgpu/*` firmware files plus the `amdgpu` kernel module
are needed in any case to boot the ReaR recovery system
on systems with AMD-GPU graphics hardware
so I think we should better have
```
# Explicitly add amdgup kernel module and its firmware files
# to boot the recovery system on AMD graphics hardware (if present)
# in particular for TCG Opal pre-boot authentication (PBA) images
# that do not include firmware files by default
# cf. https://github.com/rear/rear/issues/2474
# lspci output is expected to look similar to this:
#   00:01.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Device 98e4 (rev 81)
if type -p lspci >/dev/null && lspci | grep --quiet ' VGA .*AMD'; then
    MODULES+=( amdgpu )
    FIRMWARE_FILES+=( '*amdgpu*' )
    OPAL_PBA_FIRMWARE_FILES+=( '*amdgpu*' )
```

Reply:
Thanks for that excellent discovery! Indeed, `amdgpu` being a firmware directory will not be discovered the way I thought it would.